components/profile/personAvatar/index.test.jsx

```diff
@@ -61,6 +62,35 @@ describe("Person Avatar", () => {
     });
     expect(asFragment()).toMatchSnapshot();
   });
+
+  it("renders a webId if there is no name in the profile", async () => {
+    const nameToRemove = solidClientFns.getStringNoLocale(
+      profileThing,
+      foaf.name
+    );
+    const profileThingWithNameRemoved = solidClientFns.removeStringNoLocale(
+      profileThing,
+      foaf.name,
+      nameToRemove
+    );
+
+    const session = mockSession();
+    const SessionProvider = mockSessionContextProvider(session);
+    const { getByText } = renderWithTheme(
+      <CombinedDataProvider
+        solidDataset={profileDataset}
+        thing={profileThingWithNameRemoved}
+      >
+        <SessionProvider>
```

Comment:
SessionProvider should be on the outside of the dataset provider

Reply:
fixed